distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/restito/ChunkedContent.java

```diff
@@ -44,25 +47,14 @@
      *
      * @return the minimum headers required for sending a chunked data response
      */
-    private static Action getHeaders() {
+    public static Action getChunkedResponseHeaders() {
```

Comment:
Looking back, it seems a bit unnecessary, especially since `getRangeSupprtHeaders` only returns a single header. Made them private.